src/shared/components/controllers/Controllers.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+// should send title options and set search set option
+// controllers of all the tables
```

Comment:
delete comments

Reply:
These comments are there for explanation of the component itself, are you sure you want to delete them?